### ISSUE TITLE
Level shifter?

### ISSUE THREAD

**Issue by factus10** [zenaro147/NeoGB-Printer#2]
I hope I just missed it: how is the level shifter involved?

Is there a schematic of the entire project? If not, I’m happy to draw one.

**Comment by Raphael-Boichot** [comment]
Hello.

Basically the level shifter must be inbetween ESP pins (3.3 volts) and Game Boy serial pins (5 volts). It must be bidirectionnal and the aim is to protect the ESP from direct 5 volts inputs from Game Boy and allow GB to receive clean 5 volts from ESP. So it needs a +3.3 volts ref on ESP side and a +5 volts ref on Game Boy side (plus ground). ESPs have both.

Rafael Zenaro is preparing a PCB to ease the installation. It will be published soon. We are still trying to improve the overall design by trial-and-error.

Thanks for your interest.

Raphaël

**Comment by zenaror** [comment]
Hi @factus10 !
Thanks for your interest!

I'm so sorry! I forgot to add about the Level Shifter in the documentation.
I already added now =)

About the schematic, I already have one due to the PCB that I'm desinging.
But as soon I finished the basic draw with the mainly components, I'll add it to the documentation a first version to help other people to build one.
That's not my mainly skill and the schematic could sound a little confusing, so feel free to contribuite if you want to create a easier one =)
